product-service/PostgresClient.js

```diff
@@ -28,19 +28,71 @@ function parseTypes(row) {
   return newRow;
 }
 
-export async function query(query) {
-  log.info(`running SQL query: ${query}`);
-  let client;
-  try {
-    client = new Client(dbConfig);
-    await client.connect();
-    const queryResult = await client.query(query);
-    const rows = queryResult.rows.map(parseTypes);
-    return { ok: true, rows, rowsAffected: queryResult.rowCount };
-  } catch (error) {
-    log.error(`Error while running SQL query: ${util.inspect(error)}`);
-    return { ok: false, message: error };
-  } finally {
-    await client.end();
+export class PostgresClient {
```

Comment:
it is a bad practice to include in pr files not related to pr topic

Reply:
Thanks for such useful comments, @Romicusblr.

Reply:
I know. Just decided to combine these changes in one request (because it's course)